chainguard-dev/edu
# What are Containers? -- New conceptual article

## Type of change
New conceptual article providing an overview of what containers & container images are

### What should this PR do?
resolves #892 

### Why are we making this change?
To bring more high-level introductory content to the Academy, specifically on containers, to educate readers about Chainguard's mission

### What are the acceptance criteria? 
Article is clear and concise, follows formatting guidelines, effectively covers core topics regarding containers

### How should this PR be tested?
Check the preview to see that the article is formatted well, and provide a pass-over if any edits are needed. Specifically, the "for example" paragraph of "Structure of a Container Image" and the "Learn more" sections likely need a little bit of work :)
